src/assets/RawAsset.js

```diff
@@ -18,9 +18,12 @@ class RawAsset extends Asset {
       this.generateBundleName()
     );
 
-    return {
-      js: `module.exports=${JSON.stringify(pathToAsset)};`
-    };
+    return [
+      {
+        type: 'js',
+        value: `module.exports=${JSON.stringify(pathToAsset)};`
+      }
+    ];
```

Comment:
Did this because I ended up with `module.exports = 'somepath'`. This requires each asset to do it, is there another way?